classes/check/tasklatencycheck.php

```diff
@@ -53,89 +53,103 @@ public function get_result(): result {
         $lockstats = isset($CFG->lock_factory) && $CFG->lock_factory === \tool_lockstats\proxy_lock_factory::class;
 
         $tasks = explode(PHP_EOL, $taskconfig);
-        foreach ($tasks as $taskconfig) {
-            list($taskclass, $runtime, $startdelay, $completiondelay) = explode(',', str_replace(' ', '', $taskconfig));
-            $task = \core\task\manager::get_scheduled_task($taskclass);
 
-            // Input validation.
-            $valid = $task !== false;
+        foreach ($tasks as $taskconfig) {
+            if ($taskconfig) {
```

Comment:
This seems weird, what is it trying to catch here? If its just an empty line, I think instead we should just trim/validate more heavily beforehand.